.travis.yml

```diff
@@ -3,12 +3,42 @@ language: node_js
 node_js:
  - "8.9"
 
+services:
+  - postgresql
+
 jobs:
   include:
     - stage: test-app-waf
       install:
         - npm --prefix app/waf install
+      before_script:
+        - export PG_HOST=localhost
       script:
         - npm --prefix app/waf test
         - npm --prefix app/waf run coverage
         - npm --prefix app/waf run coveralls
+      deploy:
+        provider: script
+        on:
+          branch: master
+        script:
+          - npm --prefix app/waf start >/dev/null 2>&1 &
+          - sleep 10
+          - curl -s -X GET http://localhost:3000/adcaas/v1/ping
+          - git clone https://github.com/F5Networks/f5-adcaas-openstack.git
+          - cd f5-adcaas-openstack
+          - git checkout gh-pages
+          - HOST="localhost:3000"
+          - curl -s -X GET http://${HOST}/openapi.json > openapi.json
+          - curl -s -X GET http://${HOST}/explorer/ > index.html
+          - |
+            for file in swagger-ui.css swagger-ui-bundle.js swagger-ui-standalone-preset.js favicon-16x16.png favicon-32x32.png ; do
+              curl -s -X GET http://${HOST}/explorer/${file} > ${file}
+            done
```

Comment:
What's the benefit to write each line with '-' instead of encapsulating them into a script?

If you want the command lines to be logged, you can use `set -x`; 
If you want to quit immediately on error, you can use `set -e`

I really disagree t with them to list here, which is not suitable for the run/test locally before `git commit`. 
Just as I disagree that in our pipeline did.

Reply:
Let us reconsider it, after removing the multi-line script here.

Reply:
Move them to a script now.